### DISCUSSION TITLE
【求助】六分街的喵吉长官被肘击了，有图有真相

### DISCUSSION THREAD

**Discussion by SZI726** [share121/inter-knot#2673] [其他]
![b1a9eeafc5b124b65f1ca6d5cdc2f93d](https://github.com/user-attachments/assets/19e4c282-5c5a-4e00-9074-bc265f4e651a)
实在可恶的灰毛小子，听说他是在附近开卤香店的，有无好心人去替天行道
p.s.照片是捡到的

**Comment by LadySunBringer** [comment]
Man！

**Comment by this-til** [comment]
Man!

**Comment by Bambook-sth** [comment]
Man！

**Comment by ghost-feng** [comment]
Man!

**Comment by Obsidia-cdl** [comment]
多半是P上去的。那位录像店经营者我认识，前阵子刚来我们这边更换了营业执照，我的同事也经常来他这边租影像带，再怎么离谱也不至于对着喵吉长官来一肘。

**Comment by silverwolf0000** [comment]
绳网并非法外之地，请不要散布谣言

**Comment by woye2** [comment]
这个委托我接了！

**Comment by qqqqqq77777** [comment]
对，这个店主我听别人在六分街说好像是麦片的，难不怪呢<img src="https://bbs-static.miyoushe.com/static/2024/03/21/3248c770e7e0e7ec4a167605872b6d66_1511725123905663480.png" alt="丽都漫步-格莉丝" width="75" >
